main.py

```diff
@@ -0,0 +1,248 @@
+import json
+import telebot
+from telebot import types
+from config import token, check_text
+import requests
+import speech_recognition as sr
+import soundfile as sf
+from pydub import AudioSegment
+from backend import create_user, add_answer
+import subprocess
+import os
+
+
+
+bot = telebot.TeleBot(token)
+
+with open('text.json', encoding='utf-8') as f:
+    text = json.load(f)
+
+
+@bot.message_handler(commands=['start'])
+def send_welcome(message):
+    bot.send_message(message.chat.id, f'Hi, {message.from_user.first_name}!\n' + text["text_1"], parse_mode="Markdown", reply_markup=menu_keyboard())
+    create_user(message.from_user.username)
+
+
+@bot.message_handler(content_types=["text"])
+def step_0(message):
+    if message.text == "Practice":
+        step_1(message)
+
+    elif message.text == "Theory":
+        btn = types.InlineKeyboardButton(text='RAIN Group Sales Training', url='https://www.rainsalestraining.com/blog/4-steps-to-overcoming-sales-objections', callback_data='click1')
+        markup = types.InlineKeyboardMarkup().add(btn)
+        bot.send_message(message.chat.id, 'Let\'s study!', reply_markup=markup)
+        bot.register_next_step_handler(message, step_0)
+
+    elif message.text == "Help & settings":
+        btn = types.InlineKeyboardButton(text='Start chat with support', url='https://t.me/techh_bot_support', callback_data='click2')
+        markup = types.InlineKeyboardMarkup().add(btn)
+        bot.send_message(message.chat.id, 'You can contact help via next link:', reply_markup=markup)
+        bot.register_next_step_handler(message, step_0)
+
+    else:
+        pass
+        #bot.register_next_step_handler(message, step_0)
+
+
+def step_1(message):
+    #bot.send_message(message.chat.id, text["text_2"], parse_mode="Markdown", reply_markup=scene_keyboard())
+    bot.send_message(message.chat.id, text["text_2"], parse_mode="Markdown")
+    bot.send_message(message.chat.id, "By clicking start you agree with our rules of data collection (GDPR).", parse_mode="Markdown", reply_markup=start_scene())
+    bot.register_next_step_handler(message, step_3)
+
+
```

Comment:
2 blank lines before & after function definition (everywhere)
PyCharm, for example, may help you to avoid such problems